package.json

```diff
@@ -46,22 +46,22 @@
     "precommit": "npm run prettier -- --list-different || (echo 'Above file(s) were modified by prettier, check them before commiting' && false)"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "3.4.1",
-    "@storybook/addon-links": "3.4.1",
-    "@storybook/addon-options": "3.4.1",
-    "@storybook/addons": "3.4.1",
-    "@storybook/react": "3.4.1",
-    "@types/enzyme": "3.1.9",
+    "@storybook/addon-actions": "3.4.2",
+    "@storybook/addon-links": "3.4.2",
+    "@storybook/addon-options": "3.4.2",
+    "@storybook/addons": "3.4.2",
+    "@storybook/react": "3.4.2",
+    "@types/enzyme": "3.1.10",
     "@types/jest": "22.2.3",
-    "@types/lodash": "4.14.107",
+    "@types/lodash": "4.14.108",
     "@types/node": "9.6.5",
-    "@types/react": "16.3.11",
+    "@types/react": "16.3.12",
     "@types/react-dom": "16.0.5",
     "@types/react-router-dom": "4.2.6",
     "@types/url-search-params": "0.10.1",
-    "awesome-typescript-loader": "4.0.1",
```

Comment:
awesome-typescript-loader 5.0.0 is not compatible with current webpack version 3.11.0

Reply:
@josejulio I'm not seeing that incompatibility. The following commands all work:
* `npm run build`
* `npm start`
* `npm run test`

Reply:
I recall now, this dependency is used to allow storybooks to use typescript. But since we are removing it...